appsec-kit-backend/src/test/java/com/vaadin/appsec/backend/GitHubServiceTest.java

```diff
@@ -28,7 +28,7 @@ public class GitHubServiceTest {
     static final String TEST_RESOURCE_BOM_PATH = "/bom.json";
     static final String TEST_RESOURCE_NPM_BOM_PATH = "/bom-npm.json";
 
-    static class TestGitHubService extends GitHubService {
+    static class TestGitHubService1 extends GitHubService {
```

Comment:
Do we really need two different services? If that's so, it is not immediately clear what `1` and `2` does, let's use a more significant name.

Reply:
Renamed the service class names. The `TestGitHubServiceWithMockReleasesAndAnalysis` is used in most of the tests, it serves the releases and analysis from mock files. The `TestGitHubServiceWithMockReleases` is needed to be able to test the case when getting the analysis using a system property.